### ISSUE TITLE
Duplicate block_content

### ISSUE THREAD

**Issue by scottcorgan** [desiringgod/components-rails#21]
When using `block_content` the component renders the block_content both inside and outside the tag:

This template

<img width="255" alt="screen shot 2017-09-11 at 2 08 34 pm" src="https://user-images.githubusercontent.com/974723/30289622-c2cb8774-96fa-11e7-941f-86ad0ece830b.png">

used this way

<img width="317" alt="screen shot 2017-09-11 at 2 09 19 pm" src="https://user-images.githubusercontent.com/974723/30289640-d4491af2-96fa-11e7-9835-a20154168cc5.png">

produces this html

<img width="194" alt="screen shot 2017-09-11 at 2 08 53 pm" src="https://user-images.githubusercontent.com/974723/30289629-c998c954-96fa-11e7-93fa-6fe4baab06b2.png">

**Comment by joeosburn-desiringgod** [comment]
@benhutton I have a sneaking suspicion this is related to our new way of rendering without `ActionView#render` since this wasn't a problem before.

**Comment by benhutton** [comment]
@joeosburn-desiringgod do we not have specs covering this? Can we write some failing specs and fix it from there?

And are we seeing this anywhere in production?

**Comment by joeosburn-desiringgod** [comment]
@benhutton We aren't using this in production currently. None of the current specs use our magic component helpers. I'm going to have to create some. They are simple enough that I didn't think we needed specs for them, but assuming I'm right about my hunch, clearly I was wrong about them being so simple.

**Comment by benhutton** [comment]
@joeosburn-desiringgod yeah, there's potentially some magic going on with output buffers, concatenation, etc., right?

I wonder if it's as simple as adding a `#to_s` in the right place?

**Comment by joeosburn-desiringgod** [comment]
@benhutton There is definitely some tricky things happening with output buffers and `concat`. 

If it's as simple as adding a `#to_s`, I'd be happy. If you have an idea where, let me know. I'll work on a failing spec in the meantime.
